sambacc/commands/ctdb.py

```diff
@@ -86,6 +88,13 @@ def _ctdb_general_node_args(parser: argparse.ArgumentParser) -> None:
         "--persistent-path",
         help="Path to a persistent path for storing nodes file",
     )
+    parser.add_argument(
+        "--metadata-state",
```

Comment:
I think `--metadata-state` doesn't fully match the intended type of value which basically points to an endpoint as source of truth. Isn't it better to use `--metadata-source` or similar?

Reply:
Sure. Will update.